app.py

```diff
@@ -22,7 +22,8 @@
 slack_web_client = WebClient(token=os.environ["SLACK_BOT_TOKEN"])
 # channel = "G011U6TETPV" - Alex test channel
 # channel = "G0124A0PZ09" - pricing old standup
-channel = "G014NBSPUP8"  # Network Engineering Standup
+network_engineering_channel = "G014NBSPUP8"
+channel = os.getenv("STAND_UP_CHANNEL", network_engineering_channel)
```

Comment:
While the stand-up bot supports only one single channel, the idea is to easily set up this channel via an environment variable. I will keep NE channel to make things easier to the existent deploy